views/director-residential-address-search-choose-address.html

```diff
@@ -18,9 +18,29 @@
       
       <form action="" method="post">
         <span class="govuk-caption-l">{{directorName}}</span>
-        <h1 class="govuk-heading-l">Choose a home address</h1>
-        <!-- This is the search page for the residential address -->
-        {% from "govuk/components/input/macro.njk" import govukInput %}
+        <h1 class="govuk-heading-l">Choose an address</h1>
+
+        {% set ITEMS = [] %}
+        {% for addressOption in addresses %}
+          {% set addressObject = { 
+            id: addressOption.premises,
+            value: addressOption.premises,
+            text: addressOption.formattedAddress,
+            checked: addressOption.premises == currentPremises,
+            attributes: {
+              "data-event-id": "address-option-" + addressOption.premises
+            }
+          } %}
+          {% set ITEMS = ITEMS.concat(addressObject) %}
+        {% endfor %}
+
+        {{ govukRadios({
+          errorMessage: errors.address_array if errors,
+          id: "address_array",
+          name: "address_array",
+          value: address_array,
+          items: ITEMS
+        }) }}
 
         <div class="govuk-body">
           <a href="{{enterAddressManuallyUrl}}" class="govuk-link">Enter address manually</a>
```

Comment:
I see there's no AC but QA might likely to backlog this, 
This click is registered on the look-up page, ideally, it should be registered everywhere.